### PR TITLE
Fix/edit labeling tutorial

### PR DESCRIPTION
**Pull request by leireropl**
Edits based on issue 716. Added some format improvements and some grammar improvements, too.

### CODE REVIEW

**Comment by dvsrepo** [comment]
Would it be possible to update the images to the latest UI too? If it's not possible for all of them we could remove some of them

**Comment by leireropl** [comment]
> Would it be possible to update the images to the latest UI too? If it's not possible for all of them we could remove some of them

Yes, absolutely. I wanted to ask you how did you record/edit videos and pictures. I will text you tomorrow, then :)

File: docs/tutorials/01-labeling-finetuning.ipynb [hunk 1]
```diff
@@ -43,15 +43,17 @@
    "id": "69321d8f-d3c4-4627-96ef-de416db75181",
    "metadata": {},
    "source": [
-    "## Setup Rubrix\n",
+    "## Setup Rubrix and install tutorial dependencies\n",
```
Comment (dcfidalgo) [inline docs/tutorials/01-labeling-finetuning.ipynb]: Here i would prefer just "Setup":
```suggestion
    "## Setup\n",
```

File: docs/tutorials/01-labeling-finetuning.ipynb [hunk 2]
```diff
@@ -43,15 +43,17 @@
...
     "\n",
     "Rubrix, is a free and open-source tool to explore, annotate, and monitor data for NLP projects.\n",
     "\n",
     "If you are new to Rubrix, check out the [Github repository](https://github.com/recognai/rubrix) ⭐.\n",
     "\n",
     "If you have not installed and launched Rubrix, check the [Setup and Installation guide](../getting_started/setup&installation.rst).\n",
     "\n",
-    "Once installed, you only need to import Rubrix:"
+    "Once installed, you only need to import Rubrix. \n",
+    "\n",
```
Comment (dcfidalgo) [inline docs/tutorials/01-labeling-finetuning.ipynb]: I think we can just drop this sentence
```suggestion
```

File: docs/tutorials/01-labeling-finetuning.ipynb [hunk 3]
```diff
@@ -43,15 +43,17 @@
...
+    "In addition to this, the `transformers` and `datasets` libraries must be imported, as they are the main libraries for this tutorial:"
```
Comment (dcfidalgo) [inline docs/tutorials/01-labeling-finetuning.ipynb]:
```suggestion
    "In addition to this, the `transformers` and `datasets` libraries must be installed, which we will use to retrieve the data and the model."
```

File: docs/tutorials/01-labeling-finetuning.ipynb [hunk 4]
```diff
@@ -61,26 +63,9 @@
...
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "a906d7d1-ea0d-4158-b6dc-a386342858e6",
-   "metadata": {},
-   "outputs": [],
-   "source": [
+    "#import tutorial dependencies\n",
     "%pip install transformers -qqq\n",
     "%pip install datasets -qqq\n",
     "%pip install sklearn -qqq"
```
Comment (dcfidalgo) [inline docs/tutorials/01-labeling-finetuning.ipynb]:
```suggestion
    "# install dependencies for this tutorial\n",
    "%pip install transformers[torch] datasets sklearn -qqq"
```

File: docs/tutorials/01-labeling-finetuning.ipynb [hunk 5]
```diff
@@ -96,15 +81,15 @@
     "\n",
     "For building our fine-tuned classifier we'll be using two main resources, both available in the 🤗 Hub :\n",
     "\n",
-    "1. A **dataset** in the banking domain: `banking77`\n",
+    "1. A **dataset** in the banking domain: `banking77` [(https://huggingface.co/datasets/banking77)](https://huggingface.co/datasets/banking77)\n",
```
Comment (dcfidalgo) [inline docs/tutorials/01-labeling-finetuning.ipynb]:
```suggestion
    "1. A **dataset** in the banking domain: [banking77](https://huggingface.co/datasets/banking77)\n",
```

File: docs/tutorials/01-labeling-finetuning.ipynb [hunk 6]
```diff
@@ -96,15 +81,15 @@
...
     "\n",
-    "2. A **pre-trained sentiment classifier**: `distilbert-base-uncased-finetuned-sst-2-english`\n",
+    "2. A **pre-trained sentiment classifier**: `distilbert-base-uncased-finetuned-sst-2-english` [(https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english)](https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english)\n",
```
Comment (dcfidalgo) [inline docs/tutorials/01-labeling-finetuning.ipynb]:
```suggestion
    "2. A **pre-trained sentiment classifier**: [distilbert-base-uncased-finetuned-sst-2-english](https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english)\n",
```

File: docs/tutorials/01-labeling-finetuning.ipynb [hunk 7]
```diff
@@ -232,7 +217,7 @@
    "source": [
     "## 1. Run the **pre-trained model** over the dataset and log the predictions\n",
     "\n",
-    "As a first step, let's use the pre-trained model for predicting over our raw dataset. For this will use the handy `dataset.map` method from the `datasets` library.\n",
+    "As a first step, let's use the pre-trained model for predicting over our raw dataset. Thus, we will use the handy `dataset.map` method from the `datasets` library.\n",
```
Comment (dcfidalgo) [inline docs/tutorials/01-labeling-finetuning.ipynb]:
```suggestion
    "As a first step, let's use the pre-trained model for predicting over our raw dataset. For this, we will use the handy `dataset.map` method from the `datasets` library.\n",
```

File: docs/tutorials/01-labeling-finetuning.ipynb [hunk 8]
```diff
@@ -273,7 +258,7 @@
    "source": [
     "### Log\n",
     "\n",
-    "The following code builds a list of Rubrix records with the predictions and logs them into a Rubrix Dataset. We'll use this dataset to explore and label our first training set."
+    "The following script builds a list of Rubrix records with the predictions and logs them into a Rubrix Dataset. We'll use this dataset to explore and label our first training set."
```
Comment (dcfidalgo) [inline docs/tutorials/01-labeling-finetuning.ipynb]: script is usually associated to a proper file, i think code is fine here
```suggestion
    "The following code builds a list of Rubrix records with the predictions and logs them into a Rubrix Dataset. We'll use this dataset to explore and label our first training set."
```

File: docs/tutorials/01-labeling-finetuning.ipynb [hunk 9]
```diff
@@ -324,9 +309,9 @@
     "\n",
     "![label workflow](img/labeling_tutorial/label.svg \"label workflow\")\n",
     "\n",
-    "Rubrix provides you with a search-driven UI to annotated data, using free-text search, search filters and the Elasticsearch query DSL for advanced queries. This is most useful for sparse datasets, tasks with a high number of labels or unbalanced classes. In the standard case, we recommend you to follow the workflow below:\n",
+    "Rubrix provides you with a search-driven UI to annotated data, using **free-text search**, **search filters** and **the Elasticsearch query DSL** for advanced queries. This is super useful for sparse datasets, tasks with a high number of labels or unbalanced classes. In the standard case, we recommend you to follow the workflow below:\n",
```
Comment (dcfidalgo) [inline docs/tutorials/01-labeling-finetuning.ipynb]:
```suggestion
    "Rubrix provides you with a search-driven UI to annotated data, using **free-text search**, **search filters** and **the Elasticsearch query DSL** for advanced queries. This is especially useful for sparse datasets, tasks with a high number of labels, or unbalanced classes. In the standard case, we recommend you to follow the workflow below:\n",
```

File: docs/tutorials/01-labeling-finetuning.ipynb [hunk 10]
```diff
@@ -746,7 +731,7 @@
    "id": "7703d9a4-bfce-466f-accc-65f65a784f2d",
    "metadata": {},
    "source": [
-    "## 4. Testing the fine-tuned model\n",
+    "## 4. Test the fine-tuned model\n",
```
Comment (dcfidalgo) [inline docs/tutorials/01-labeling-finetuning.ipynb]: Below you used the present participle ("Adding", "Training"), so i would use it here, too
 ```suggestion
    "## 4. Testing the fine-tuned model\n",
```

File: docs/tutorials/01-labeling-finetuning.ipynb [hunk 11]
```diff
@@ -1137,13 +1122,13 @@
    "id": "529d3e6b-0684-47db-ab69-ff705945363f",
    "metadata": {},
    "source": [
-    "## Wrap-up\n",
+    "## Summary\n",
     "\n",
-    "In this tutorial, you've learnt to build a training set from scratch with the help of a pre-trained model, performing two iterations of `predict` > `log` > `label`. \n",
+    "In this tutorial, you have learnt how to build a training set from scratch with the help of a pre-trained model, performing two iterations of `predict` > `log` > `label`. \n",
```
Comment (dcfidalgo) [inline docs/tutorials/01-labeling-finetuning.ipynb]:
```suggestion
    "In this tutorial, you learned how to build a training set from scratch with the help of a pre-trained model, performing two iterations of `predict` > `log` > `label`. \n",
```

**Review by dcfidalgo — CHANGES_REQUESTED** [review]
Thanks a lot! I made a few inline suggestions. Also, could you do a rebase on the master branch to avoid merge conflicts? Maybe it would be best to first merge the other open PR, and then this one, since both will modify the same file.

**Comment by leireropl** [comment]
I think I committed all changes. Will do the images and rebasing next day!

**Review by dcfidalgo — APPROVED** [review]

**Pull request merged**